build-tools/packages/build-cli/docs/typetestDetails.md

```diff
@@ -152,7 +152,7 @@ Given the version 2.0.0-internal.2.0.0:
 | `previousMinor`        | 2.0.0-internal.2.0.0                          |
 | `^previousMajor`       | >=2.0.0-internal.1.0.0 <2.0.0-internal.2.0.0  |
 | `^previousMinor`       | >=2.0.0-internal.2.0.0 <2.0.0-internal.3.0.0  |
-| `~previousMajor`       | >=2.0.0-internal.1.0.0 <2.0.0-internal.1.1.0  |
+| `~previousMajor`       | >=2.0.0-internal.1.0.0 <2.0.0-internal.2.0.0  |
```

Comment:
The original was correct on this one. The previousMajor is the same as [this one](https://github.com/microsoft/FluidFramework/pull/13053/files#r1029670487) so the same logic applies.

Reply:
Reverting too.